Unused variable in the installation script (install.sh) for GNU/Linux distributions

🟡 **Checking the Tailscale installation script [(install.sh)](https://tailscale.com/install.sh) for GNU/Linux distributions with [ShellCheck](https://github.com/koalaman/shellcheck) shows that there is a `PACKAGE_NAME="tailscale"` variable defined but not used on line 537 :**

<img width="3072" height="3072" alt="Image" src="https://github.com/user-attachments/assets/34f13986-8c08-4a74-905c-06cdf427ccdb" />

----

<img width="3072" height="3072" alt="Image" src="https://github.com/user-attachments/assets/f55571bb-54d0-419c-b7fa-73dc14ab668a" />

**[ShellCheck SC2034](https://www.shellcheck.net/wiki/SC2034)
ShellCheck version : [0.11.0](https://github.com/koalaman/shellcheck/releases/tag/v0.11.0)**

--- a/scripts/installer.sh
+++ b/scripts/installer.sh
@@ -534,7 +534,6 @@ main() {
 	[ "$VERSION" != "" ] && OSVERSION="$OSVERSION $VERSION"
 
 	# Prepare package name with optional version
-	PACKAGE_NAME="tailscale"
 	if [ -n "$TAILSCALE_VERSION" ]; then
 		echo "Installing Tailscale $TAILSCALE_VERSION for $OSVERSION, using method $PACKAGETYPE"
 	else

